.github/dependabot.yml

```
@@ -20,3 +20,5 @@ updates:
       go-dependencies:
         patterns:
           - "*"
+        exclude-patterns:
+          - "k8s.io/*"
```

```suggestion
          - "k8s.io/*"
          - "sigs.k8s.io/*"
```

I think think one also needs to be added to avaoid the controller runtime to be upgraded